batch.com/headers-bundle/1.0/config/packages/batch_headers.yaml

```diff
@@ -0,0 +1,19 @@
+batch_headers:
+    headers:
+        # Define your response headers here. See the examples below.
+        #
+        # Apply a CSP on all the responses:
+        #
+        #     - Content-Security-Policy: default-src 'self'
+        #
+        # Allow your API to be requested from all origins:
+        #
+        #     - name: Access-Control-Allow-Origin
+        #       value: "*"
+        #       condition: request.getPathInfo() matches '^/api'
+        #
+        # Always cache images:
+        #
+        #     - name: Cache-Control
+        #       value: max-age=31536000, public
+        #       condition: response.headers.get('Content-Type') matches '^image/'
```

Comment:
Examples should be in your bundle's readme. Not as comments. 

Please remove these and add a link to your bundle's docs. 

(Maybe, it makes sense to uncomment one of them an remove the others)

Reply:
Sorry for the delay… I've just fixed this. 🙂